examples/kubernetes/connectivity-check/Makefile

```diff
@@ -48,6 +49,11 @@ $(QUARANTINE_OUT): $(SRC) $(SERVERS_OUT)
 	$(QUIET)$(CUE) -t component=all -t quarantine=true dump > $@
 	@cat $(SERVERS_OUT) >> $@
 
+$(TOOLS_OUT): $(SRC) $(SERVERS_OUT)
+	@echo '# Automatically generated by Makefile. DO NOT EDIT' > $(TOOLS_OUT)
+	$(QUIET)$(CUE) -t component=all -t type=tool dump > $@
```

Comment:
as highlighted by @fristonio in another PR, >> should be used to avoid overriding first line.